cmd/otelcol/main.go

```diff
@@ -52,7 +52,7 @@ func runInteractive(params service.Parameters) error {
 
 	err = app.Start()
 	if err != nil {
-		return errors.Wrap(err, "application run finished with error: %v")
+		return errors.Wrap(err, "application run finished with error")
```

Comment:
use fmt.errorf instead of wrap?

Reply:
My description might have been ambiguous, there's nothing wrong with this line besides the fact that the "%v" doesn't actually do anything in this context since `errors.Wrapf`isn't used. 

The err gets properly appended to the log when it is executed.

Reply:
In golang 1.13 they added `fmt.errorf` https://golang.org/pkg/fmt/#Errorf which is a replacement for `errors.Wrap`

Reply:
Understood. Done. Thank you!